languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/added.txt

```diff
@@ -12784,3 +12784,32 @@ Stretching	Stretching	SUB:NOM:SIN:NEU
 Stretchings	Stretching	SUB:GEN:SIN:NEU
 Stretching	Stretching	SUB:DAT:SIN:NEU
 Stretching	Stretching	SUB:AKK:SIN:NEU
+Heatpipe	Heatpipe	SUB:NOM:SIN:FEM
+Heatpipe	Heatpipe	SUB:GEN:SIN:FEM
+Heatpipe	Heatpipe	SUB:DAT:SIN:FEM
+Heatpipe	Heatpipe	SUB:AKK:SIN:FEM
+Heatpipes	Heatpipe	SUB:NOM:PLU:FEM
+Heatpipes	Heatpipe	SUB:GEN:PLU:FEM
+Heatpipes	Heatpipe	SUB:DAT:PLU:FEM
+Heatpipes	Heatpipe	SUB:AKK:PLU:FEM
+Bidet	Bidet	SUB:NOM:SIN:FEM
+Bidet	Bidet	SUB:GEN:SIN:FEM
```

Comment:
Genitiv ist "Bidets" https://www.duden.de/rechtschreibung/Bidet

Bitte immer mit dem Duden abgleichen. Danke!

Reply:
Dazu kann ich sagen, dass ich in dem Duden nachgeschlagen habe. Ich weiß nur nicht, warum ich Bidet jetzt weiblich eingetragen wurde. in meiner Added.txt war es denitiv nicht weiblich eingetragen. Ich glaube, ich habe aus der falschen Datei kopiert.

Ich danke dir, dass du den Fehler bemerkt hast.

Reply:
Laut Duden ist es neutrum.

Reply:
wurde jetzt alles korrigiert. Der Notepad++ hat mir sogar bestätigt, dass ich aus der falschen Copy und Paste gemacht habe. Deswegen ist es schon immer gut gewesen, wenn ein zweiter drüber schaut. 

Deswegen noch einmal: Ich danke dir sehr, dass du darüber geschaut hast.